frontend/src/lib/components/CategoryItem.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+    // import CircularProgressBar from './CircularProgressBar.svelte';
+    export let title;
+    export let categoryId;
+
+    const progress = 1;
+</script>
+
+<div class="bg-white rounded-xl p-1 flex justify-between" style="box-shadow: rgba(0, 0, 0, 0.15) 2.4px 2.4px 3.2px">
+    <a href={`categories/${categoryId}/habits`} class="flex w-full">
+        <img alt="routine icon" src="../../src/img/routineIcon{categoryId}.png" class="object-scale-down">
```

Comment:
When you add new categories their id increases and there are only 3 images, therefore no image is available and 404 gets thrown in the console.

Reply:
they will repeat for now